account_payment_order/models/account_payment_order.py

```diff
@@ -25,7 +25,7 @@ def domain_journal_id(self):
             return [('id', 'in', jrl_ids)]
 
     name = fields.Char(
-        string='Number', readonly=True, copy=False)  # v8 field : name
+        string='Number', readonly=False, copy=False)  # v8 field : name
```

Comment:
This change seems unrelated to the scope of the PR.

Reply:
Oups, it's a mistake. That was only for test purposes. I Changed it back in my last commit https://github.com/OCA/bank-payment/pull/658/commits/2e1b8baa27bf83493f14237de0248ccda14ca931